src/textual/demo.py

```diff
@@ -313,7 +294,7 @@ def add_note(self, renderable: RenderableType) -> None:
         self.query_one(TextLog).write(renderable)
 
     def compose(self) -> ComposeResult:
-        example_css = "\n".join(Path(self.css_path[0]).read_text().splitlines()[:50])
+        example_css = "\n".join(Path(self.css_path[0]).read_text().splitlines())
```

Comment:
The `"\n".join(...splitlines())` was only used because we wanted to be able to display full lines (otherwise slicing would be difficult).

If we are not slicing, and instead we display the full CSS, no need to split and join.

Reply:
@rodrigogiraoserrao I see, done.

Reply:
Thanks for the quick turnaround.